crates/matrix-sdk-crypto-ffi/uniffi.toml

```diff
@@ -0,0 +1,2 @@
+[bindings.swift]
+module_name = "MatrixSDKCrypto"
```

Comment:
Without this config file used by uniffi-bindgen, all names of the outputs are simply olm.xcframework, olm.swift etc, which is both too cryptic as well as legacy (as we move towards vodozemac)